Gemfile.lock

```diff
@@ -144,14 +151,150 @@ GEM
       logger
     faraday-net_http (3.4.0)
       net-http (>= 0.5.0)
+    fission (0.5.0)
+      CFPropertyList (~> 2.2)
+    fog (1.38.0)
+      fog-aliyun (>= 0.1.0)
+      fog-atmos
+      fog-aws (>= 0.6.0)
+      fog-brightbox (~> 0.4)
+      fog-cloudatcost (~> 0.1.0)
+      fog-core (~> 1.32)
+      fog-dynect (~> 0.0.2)
+      fog-ecloud (~> 0.1)
+      fog-google (<= 0.1.0)
```

Comment:
this brings in a lot of dependencies for the ability to generate an s3 signed url, can you try using the [fog-aws gem](https://github.com/fog/fog-aws) instead to reduce the bloat here? (in the Gemfile). btw I assume this is a straightforward swap and replace, if this takes a lot more work please let me know.
```
- gem "fog", "~> 1.38"
+ gem "fog-aws"
```